pkg/scene/widget_align_system.go

```diff
@@ -0,0 +1,250 @@
+package scene
+
+import (
+	"github.com/reearth/reearth-backend/pkg/id"
+)
+
+// WidgetAlignSystem is the layout structure of any enabled widgets that will be displayed over the scene.
+type WidgetAlignSystem struct {
+	inner WidgetZone
+	outer WidgetZone
+}
+
+// WidgetZone is the structure of each layer (inner and outer) of the align system.
+type WidgetZone struct {
+	left   WidgetSection
+	center WidgetSection
+	right  WidgetSection
+}
+
+// WidgetSection is the structure of each section of the align system.
+type WidgetSection struct {
+	top    WidgetArea
+	middle WidgetArea
+	bottom WidgetArea
+}
+
+// WidgetArea has the widgets and alignment information found in each part area of a section.
+type WidgetArea struct {
+	widgetIds []*id.WidgetID
+	align     string
+}
+
+type WidgetLocation struct {
+	Zone    string
+	Section string
+	Area    string
+}
+
+const (
+	AlignStart  = "start"
+	AlignCenter = "center"
+	AlignEnd    = "end"
+)
+
+var Areas = []string{
+	"top",
+	"middle",
+	"bottom",
```

Comment:
```suggestion
	AlignTop,
	AlignMiddle,
	AlignBottom,
```

We have all these constants, but they're not being used!

Reply:
I am using these in the FindWidgetLocation method's for loops' ranges.